index.html

```diff
@@ -1238,6 +1238,192 @@
 <section id="converting-html-to-other-formats">
 <h2>Converting Microdata to other formats</h2>
 
+<section id="jsonLD">
+<h3>JSON-LD</h3>
+
+  <p>Given a list of nodes <var>nodes</var> in a <code>Document</code>, a user agent must
+  run the following algorithm to extract the Microdata from those nodes into a JSON-LD:</p>
+
+  <ol>
+
+   <li><p>Let <var>result</var> be an empty object.</p></li>
+
+   <li><p>Let <var>items</var> be an empty array.</p></li>
+
+   <li><p>For each <var>node</var> in <var>nodes</var>, check if the element is a
+   <a>top-level Microdata item</a>, and if it is then <a>get the object</a> for that element and add it to <var>items</var>.</p></li>
+
+   <li><p>Add an entry to <var>result</var> called "<code>items</code>" whose
+   value is the array <var>items</var>.</p></li>
```

Comment:
Having a top-level entry "items" is not useful, unless "items" is aliased to "@graph". "@graph" is used to contain top-level node definitions in JSON-LD. But, if there is no top-level `@context` (see note elsewhere), then perhaps result is simply an array.